src/components/pill/index.js

```diff
@@ -0,0 +1,123 @@
+import { html, css, LitElement, unsafeCSS } from 'lit';
+import styles from './styles.scss';
+
+const props = {
+  disabled: 'data-vl-disabled',
+  type: 'data-vl-type',
+  closable: 'data-vl-closable',
+  checkable: 'data-vl-checkable',
+};
+
+const { disabled, type, closable, checkable } = props;
+
+export class VlPill extends LitElement {
+  static get EVENTS() {
+    return {
+      close: 'close',
+      check: 'check',
+    };
+  }
+
+  static get styles() {
+    return [
+      css`
+        ${unsafeCSS(styles)}
+      `,
+    ];
+  }
+
+  static get properties() {
+    return {
+      [disabled]: { type: Boolean },
+      [type]: { type: String },
+      [closable]: { type: Boolean },
+      [checkable]: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this[type] = '';
+    this[disabled] = false;
+    this[closable] = false;
+    this[checkable] = false;
+  }
+
+  checked() {
+    const checkbox = this._checkbox;
+    if (checkbox) {
+      return !!checkbox.checked;
+    }
+  }
+
+  setChecked() {
+    const checkbox = this._checkbox;
+    if (checkbox) {
+      checkbox.checked = checked;
+      this._checked();
+    }
+  }
+
+  _checkbox() {
+    return this.shadowRoot.querySelector('#checkbox');
```

Comment:
Vermijd de queryselector in uw component zelf als het kan, dit kan opgelost worden met de reference decorator van Lit. https://lit.dev/docs/api/directives/#ref

Reply:
Fixed.